src/pages/AccommodationsByLocation/AccommodationsByLocation.tsx

```diff
@@ -0,0 +1,18 @@
+import { useLocation } from "react-router-dom";
+import styles from "./AccommodationsByLocation.module.css";
+
+interface CustomState {
+  location: string;
+}
+
+export const AccommodationsByLocation = () => {
+  const location = useLocation();
+  const state = location.state as CustomState;
```

Comment:
Wouldn't be a bad idea to add a `<Redirect />` if `location.state` is `undefined`